assets/tests/components/detours/detoursPage.addDetour.test.tsx

```diff
@@ -13,6 +13,10 @@ import getTestGroups from "../../../src/userTestGroups"
 
 jest.mock("../../../src/userTestGroups")
 
+jest.mock("../../../src/hooks/useAllDetours", () => ({
+  useAllDetours: jest.fn(() => {}),
+}))
```

Comment:
note: we should mock the fetch, not the hook, this makes sure we test anything related to promises resolving

We should also be settings mocks in `beforeEach` and not in 
```suggestion
import { fetchDetours } from "../../../src/api"

import { neverPromise } from "../../testHelpers/mockHelpers"

jest.mock("../../../src/api")

beforeEach(() => {
    jest.mocked(fetchDetours).mockReturnValue(neverPromise())
})
```

Because this is prevalent in Skate's existing test code, I decided to write up my thoughts on this in our notion, feel free to add or modify or ask any questions, https://www.notion.so/mbta-downtown-crossing/Code-and-Coding-Norms-7f7a302480dd4c5e8c8e7274d5a7ca49?pvs=4#0d1e961a6b8946c69a9f2424b6cfb09b